openassessment/xblock/team_workflow_mixin.py

```diff
@@ -107,3 +108,32 @@ def get_team_workflow_status_counts(self):
         )
         num_submissions = sum(item['count'] for item in status_counts)
         return status_counts, num_submissions
+
+    def get_team_workflow_cancellation_info(self, team_submission_uuid):
+        """
+        Returns cancellation information for a particular team submission.
+
+        :param team_submission_uuid: The team_submission identifier associated with the
+        sumbission to return information for.
+        :return: The cancellation information, or None if the team submission has
+        not been cancelled.
+        """
+        cancellation_info = team_workflow_api.get_assessment_workflow_cancellation(
+            team_submission_uuid
+        )
+        if not cancellation_info:
+            return None
+
+        # Add the username of the staff member who cancelled the submission
+        cancellation_info['cancelled_by'] = self.get_username(cancellation_info['cancelled_by_id'])
+
+        # Add the date that the workflow was cancelled (in preference to the serialized date string)
+        del cancellation_info['created_at']
```

Comment:
Why are we clearing out the 'created_at' ?

Reply:
copying from the workflow_mixin, so I'm not 100% sure.